app/controllers/loading/AddExtraInformationYesNoController.scala

```diff
@@ -34,7 +34,7 @@ import scala.concurrent.{ExecutionContext, Future}
 class AddExtraInformationYesNoController @Inject() (
   override val messagesApi: MessagesApi,
   implicit val sessionRepository: SessionRepository,
-  navigator: Navigator,
+  navigator: LoadingNavigator,
```

Comment:
What is the difference between this approach and using the annotation approach with a bound LoadingNavigator?

Reply:
Also interested in this but think we need to be consistent across both tickets. Ive noticed the other PR uses the annotation approach. @kenneth-rayner might be worth speaking with @BledarFeratiEY to keep both navigation tickets in line